### PR TITLE
feat(assets): add sbanken logo svgs

### PR DESCRIPTION
**Pull request by AadneRo**
The svgs have been optimised, but are still a bit large.

### CODE REVIEW

File: packages/dnb-eufemia/assets/images/sbanken/sbanken-horizontal.svg
Comment (tujoworker) [inline packages/dnb-eufemia/assets/images/sbanken/sbanken-horizontal.svg:1]: This one looks not right. Is it just GitHub that cuts it off?

Comment (AadneRo) [reply packages/dnb-eufemia/assets/images/sbanken/sbanken-horizontal.svg:1]: What looks wrong? As far as I can tell this is what it has looked like the whole time. I can change the name to sbanken-compact-horizontal.svg, that is more descriptive

Comment (tujoworker) [reply packages/dnb-eufemia/assets/images/sbanken/sbanken-horizontal.svg:1]: yeah, you are right.

**Review by tujoworker — APPROVED** [review]
Nice 📚

**Pull request merged**